mmv1/templates/terraform/examples/iam_access_boundary_policy_basic.tf.erb

```diff
@@ -0,0 +1,49 @@
+resource "google_project" "project" {
+  project_id      = "tf-test%{random_suffix}"
+  name            = "tf-test%{random_suffix}"
+  org_id          = "<%= ctx[:test_env_vars]['org_id'] %>"
+  billing_account = "<%= ctx[:test_env_vars]['billing_account'] %>"
+}
+
+resource "google_access_context_manager_access_level" "test-access" {
+  parent = "accessPolicies/${google_access_context_manager_access_policy.access-policy.name}"
+  name   = "accessPolicies/${google_access_context_manager_access_policy.access-policy.name}/accessLevels/tf_test_chromeos_no_lock%{random_suffix}"
+  title  = "tf_test_chromeos_no_lock%{random_suffix}"
+  basic {
+    conditions {
+      device_policy {
+        require_screen_lock = true
+        os_constraints {
+          os_type = "DESKTOP_CHROME_OS"
+        }
+      }
+      regions = [
+        "CH",
+        "IT",
+        "US",
+      ]
+    }
+  }
+}
+
+resource "google_access_context_manager_access_policy" "access-policy" {
+  parent = "organizations/123456789"
```

Comment:
This will be used to create an actual GCP resource, so hard-coding to a fake organization ID won't work but you should be able to access the CI org_id similar to how the project accesses it above

Reply:
I believe that our ACM tests are run in serial due to how ACM access policies are essentially singletons on an organization. This means our ACM tests are handwritten to enforce that they are not run in parallel. Something similar will likely need to be done for these tests as they require setting up access policies in the org.